config/locales/de.yml

```diff
@@ -95,6 +95,8 @@ de:
         duration: "Dauer in Tagen"
         metric: "Metrik"
         initial_value: "Anfangswert"
+        min_players_per_team: "Min. Anzahl Spieler pro Team"
```

Comment:
maybe call this "minimale Teamgröße" and "maximale Teamgröße"? sounds a bit more natural